admin/admin-filters-columns.php

```diff
@@ -1,7 +1,4 @@
 <?php
```

Comment:
Strange because it makes a WPCS error but I haven't this error on local on my branches where I haven't the `@package Polylang` too